feature/notification/src/main/java/org/sopt/official/feature/notification/all/NotificationActivity.kt

```diff
@@ -103,106 +107,132 @@ class NotificationActivity : AppCompatActivity() {
-                        LazyColumn(
+                    },
+                    containerColor = SoptTheme.colors.background
+                ) { innerPadding ->
+                    Column(
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .padding(innerPadding)
+                    ) {
+                        Row(
+                            horizontalArrangement = Arrangement.spacedBy(10.dp),
                             modifier = Modifier
-                                .fillMaxSize()
-                                .padding(innerPadding)
+                                .padding(start = 20.dp, top = 12.dp, bottom = 10.dp)
                         ) {
-                            items(notifications.itemCount) {
-                                val item = notifications[it]
-                                Column(modifier = Modifier
-                                    .fillMaxWidth()
-                                    .height(100.dp)
-                                    .clickable {
-                                        context.startActivity(navigator.getNotificationDetailActivityIntent(item?.notificationId.orEmpty()))
+                            NotificationCategory.entries.forEach { notification ->
+                                NotificationCategoryChip(
+                                    category = notification.category,
+                                    isSelected = state.notificationCategory == notification,
+                                    onClick = {
+                                        viewModel.updateNotificationCategory(category = notification)
                                     }
-                                    .background(
-                                        if (item?.isRead == true) {
-                                            SoptTheme.colors.onSurface800
-                                        } else {
-                                            SoptTheme.colors.background
-                                        }
-                                    )
-                                    .padding(
-                                        horizontal = 20.dp,
-                                        vertical = 16.dp
-                                    )) {
-                                    Row(
-                                        modifier = Modifier.fillMaxWidth(),
-                                        horizontalArrangement = Arrangement.SpaceBetween,
-                                        verticalAlignment = Alignment.CenterVertically
+                                )
+                            }
+                        }
+
+                        if (notifications.itemCount > 0) {
+                            LazyColumn {
+                                items(notifications.itemCount) {
+                                    val item = notifications[it]
```

Comment:
items를 통해서 리스트 내부 값에도 접근이 가능할거에요.
아래처럼 사용하는건 어떨까요?
```suggestion
                                items(notifications) { notification ->
```

Reply:
```kotlin
            val notifications = viewModel.notifications.collectAsLazyPagingItems()
```
이거 paging item이어서 저렇게 써야해요

Reply:
맞아요... 왜 `item`에 저장하지? 싶었는데 paging item이라서 `notifications`를 바로 넣는게 안되더라구요😅